docs/README.md

```diff
@@ -9,7 +9,7 @@ the SDK are available by topic:
 * [Client](client.md)
 * [Classifications](classifications.md)
 * [Collaborations](collaborations.md)
-* [Collaboration Whitelist](collaboration-whitelist.md)
```

Comment:
Thanks @sujaygarlanka. Maybe instead of doing the text replace changes to the docs folder I would just add notes to specific classes and functions that were renamed, thoughts? Something to note is that the docs are automatically generated from the code and this process will need to be updated anyway once we migrated full to TypeScript.

Reply:
@mhagmajer That works as well. I would also see if there is a good way to communicate in the code the all whitelist methods are deprecated, like adding a deprecated decorator or something of that sort to `whitelist` methods.

Reply:
Good idea, I added the `@deprecated` annotations to:
- `collaborationWhitelist` on `BoxClient`
- `getWhitelistedDomain` and `getAllWhitelistedDomains` on `CollaborationAllowlist`

Reply:
@sujaygarlanka I updated  `docs/collaboration-whitelist.md` but left jsdocs as is - these are generated docs and we should regenerate them from TypeScript in a separate PR